### DISCUSSION TITLE
Persist Stores

### DISCUSSION THREAD

**Discussion by philippedasilva-orizone** [vuejs/pinia#465] [Help and Questions] [answered]
Hi,

What would be one's approach to implement store persistance either using window.localStorage, sessionStorage, IndexedDb or anything else?...

I used vuex-persist or vuex-persiststates vuex plugins and I'd like to implement something similar but with the KISS principles Pinia seems to try to follow.

Store persistence is the sole feature that I lack to move from a really verbose Vuex 4.x current state to Pinia.

Thanks

**Comment by posva** [comment, marked as answer]
https://github.com/posva/pinia/issues/309#issuecomment-740574516

**Reply by philippedasilva-orizone** [reply]
Thanks for the quick reply, I couldn't find answers reading the discussion board (blame me or Github discussion search... 😏). Sorry for the inconvenience...
